src/level_gen.cpp

```diff
@@ -225,9 +229,9 @@ static Entity makeStopSign(Engine &ctx, const MapVector2 &p1) {
     float y1 = p1.y;
 
     auto stop_sign = ctx.makeRenderableEntity<PhysicsEntity>();
-    ctx.get<Position>(stop_sign) = Vector3{.x = x1 - ctx.data().mean.x, .y = y1 - ctx.data().mean.y, .z = 0.5};
+    ctx.get<Position>(stop_sign) = Vector3{.x = x1 - ctx.data().mean.x, .y = y1 - ctx.data().mean.y, .z = 1};
     ctx.get<Rotation>(stop_sign) = Quat::angleAxis(0, madrona::math::up);
-    ctx.get<Scale>(stop_sign) = Diag3x3{.d0 = 0.2, .d1 = 0.2, .d2 = 0.5};
+    ctx.get<Scale>(stop_sign) = Diag3x3{.d0 = 0.2, .d1 = 0.2, .d2 = 1};
```

Comment:
What does this do? Why are you changing the stop_sign scale from 0.5 to 1.0?

Reply:
It was necessary to increase the heights of stop sign so that Lidar rays can intersect with the stop signs. Simply offsetting on the z axis would have made them non collidable.